Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY lookout/__init__.py lookout/__init__.py
 COPY lookout/__main__.py lookout/__main__.py
 COPY setup.py setup.py
 COPY requirements.txt requirements.txt
+COPY README.md README.md
```

Comment:
We need to change the way we build this, otherwise such bugs will continue to happen.
Looks like we need to copy everything and remove those which is not relevant (e.g., the lookout sdk submodule).

Reply:
ok!

Reply:
we can use `.dockerignore` file for this.